checkOpenPMD_h5.py

```diff
@@ -74,7 +75,27 @@ def get_attr(f, name):
         return(True, f.attrs[name])
     else:
         return(False, None)
-
+        
+def get_extensions(f, v):
+    """
+    Get a dictionary which maps each extension name to a bool whether it is enabled in the file
+    """
+    valid, extensionIDs = get_attr(f, "openPMDextension")
+    result = {ext: False for ext in ext_list.keys()}
+    if valid:
+        enabledExtMask = 0
+        for extension in ext_list.keys():
+            if (ext_list[extension] & extensionIDs) == ext_list[extension]:
```

Comment:
can you please add a little inline comment here?
in the last review I got the feedback that seeing this is a bitmask operation (nor what it means) is not obvious: https://github.com/openPMD/openPMD-viewer/blob/ce923f80ca82bffcd6609ab2ec124bbceb667055/opmd_viewer/openpmd_timeseries/data_reader/params_reader.py#L60-L67